Fixes for CentOS 9

rebtoor opened the pull request:
* retrieve distribution and its versions using ansible facts
* new name of guestfs-tools package
* absolute path for vbmc
* loosened the check of distribution_version for iptables task on older releases
* implemented a check for the existence of routing on OC routers
* fixed a typo in a command line option

Commit:
Fixes for CentOS 9

rebtoor committed:
Uses ansible facts to retrieve distribution and its version

cjeanner commented on roles/builder/tasks/virtualbmc.yaml:
```diff
@@ -72,7 +72,7 @@
 
     - name: Manage iptables on older centos release
       when:
-        - ansible_facts['distribution_version'] is version('8.0', 'lt', strict=True)
```
really? I'm not sure removing the "strict" here has anything to do, we don't want iptables things for 8 nor upper versions.

cjeanner replied:
"strict" is useless actually.

rebtoor committed:
Removed strict=False from version comparison since it's the default behaviour

rebtoor committed:
added ceph in tripleo_repos_repos variable

rebtoor committed:
Merge branch 'cjeanner:master' into master

rebtoor committed:
Select guestfs-tools correct package name for different OS versions

rebtoor committed:
Fixed versions comparison check

rebtoor committed:
Added gpg keys end enabled gpg check to repos

rebtoor committed:
Fixed vip config for standalone environment with HA enabled.

On standalone env with ha enabled, galera failed to start.

See this [1] for major explanation, especially this [2] comment.

[1] https://bugs.launchpad.net/tripleo/+bug/1876441
[2] https://bugs.launchpad.net/tripleo/+bug/1876441/comments/9

rebtoor committed:
Merge branch 'cjeanner:master' into master
